src/components/StoreWonderCart.js

```diff
@@ -0,0 +1,32 @@
+import { Button, Card, Title } from "@newfold/ui-component-library";
+import { __ } from "@wordpress/i18n";
+
+export function StoreWonderCart({ yithPluginsMap, id }) {
+  return (
+    <Card id={yithPluginsMap.get(id).title}>
+      <Card.Content className={"nfd-flex nfd-flex-col nfd-gap-3"}>
+        <div className={"nfd-flex nfd-flex-row nfd-gap-3 nfd-items-center"}>
+          <img
+            src={yithPluginsMap.get(id).image}
+            className="nfd-w-12 nfd-text-[--nfd-ecommerce-text-dark]"
+          />
+          <Title size="4" className="nfd-leading-normal nfd-my-4">
+            {__(yithPluginsMap.get(id).name, "wp-module-ecommerce")}
+          </Title>
+        </div>
+        <span>{yithPluginsMap.get(id).desc}</span>
```

Comment:
can I know if this is digit or we're missing translations here ?

Reply:
updated translations